tests/media/video/test_video_import.py

```
@@ -0,0 +1,335 @@
+        # Mock frame cleaning to fail
+        with patch.object(VideoImportService, '_ensure_frame_cleaning_available') as mock_frame_cleaning:
+            mock_frame_cleaning.return_value = (False, None, None)
+            
+            # Mock video creation to fail
+            with patch('endoreg_db.models.VideoFile.create_from_file_initialized') as mock_create_video:
+                mock_create_video.side_effect = Exception("Simulated creation error")
+                
+                service = VideoImportService()
+                
+                # Import should fail gracefully
+                with self.assertRaises(Exception):
```

Ruff (0.14.1)

Do not assert blind exception: `Exception`

(B017)